godot-macros/src/lib.rs

```diff
@@ -679,9 +679,36 @@ pub fn derive_godot_class(input: TokenStream) -> TokenStream {
 /// # Constants and signals
 ///
 /// Please refer to [the book](https://godot-rust.github.io/book/register/constants.html).
+///
+/// # Multiple inherent impl blocks
```

Comment:
```suggestion
/// # Multiple inherent `impl` blocks
```

Also, it's not part of the table of contents in the beginning. Alternatively, you can replace the existing ToC with a mention of _Sections_ in the sidebar, maybe in bold, something along:

> **For a table of contents, consult the _Sections_ label in the sidebar on the left.**

Reply:
I've added it to the TOC